plans/dht/test/find_peers.go

```diff
@@ -27,19 +32,24 @@ func LookupPeers(runenv *runtime.RunEnv) {
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 
-	// moot assignment to pacify the compiler. We need to merge the configurable
-	// bucket size param upstream.
-	_ = bucketSize
+	/// --- Warm up
 
-	h, err := libp2p.New(context.Background())
+	node, err := libp2p.New(context.Background())
 	if err != nil {
 		runenv.Abort(err)
 		return
 	}
+	myNodeID := node.ID()
 
-	runenv.Message("I am %s with addrs: %v", h.ID(), h.Addrs())
+	runenv.Message("I am %s with addrs: %v", myNodeID, node.Addrs())
+
+	// TODO enable/disable random-walk based on test parameter
```

Comment:
Figuring out how to do this. Threads

- https://discuss.libp2p.io/t/how-to-enable-disable-dht-random-walks/336
- https://github.com/libp2p/go-libp2p/issues/222

Reply:
Solved! ✅